#Serviced by /gaussian_process/start_process
#REQ cloud: full path  to dir containing hand.pcd and obj.pcd  to load, if empty
#calls pacman_vision get cloud in hand to get point clouds from vision.
string cloud_dir
---
